mycore-webtools/src/main/resources/xsl/msie-warning-3.xsl

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
```

Comment:
Is this still necessary in 2023 - with IE nearly dead?

Reply:
you never know, currently its used in the skeleton, thats why i migrated it.